hermes-consumers/src/main/java/pl/allegro/tech/hermes/consumers/consumer/rate/ConsumerRateLimitSupervisor.java

```diff
@@ -19,11 +19,10 @@ public class ConsumerRateLimitSupervisor implements Runnable {
 
     private final Set<ConsumerRateLimiter> consumerRateLimiters = Collections.newSetFromMap(new ConcurrentHashMap<>());
 
-    public ConsumerRateLimitSupervisor(ConfigFactory configFactory) {
-        int period = configFactory.getIntProperty(Configs.CONSUMER_RATE_LIMITER_SUPERVISOR_PERIOD);
+    public ConsumerRateLimitSupervisor(int rateLimiterSupervisorPeriod) {
         ThreadFactory threadFactory = new ThreadFactoryBuilder().setNameFormat("rate-limit-supervisor-%d").build();
         Executors.newSingleThreadScheduledExecutor(threadFactory)
-                .scheduleAtFixedRate(this, period, period, TimeUnit.SECONDS);
+                .scheduleAtFixedRate(this, rateLimiterSupervisorPeriod, rateLimiterSupervisorPeriod, TimeUnit.SECONDS);
```

Comment:
Shouldn't it be `.scheduleAtFixedRate(this, rateLimiterSupervisorPeriod, TimeUnit.SECONDS);` ?

Reply:
this function look like this so I think that was intentional
<img width="549" alt="Zrzut ekranu 2022-05-27 o 12 37 46" src="https://user-images.githubusercontent.com/76775507/170684074-c03fdaeb-8544-429e-933e-091c63604793.png">

Reply:
I didn't notice that there was also a duplicate in the previous version